cmd/federation-controller/main.go

```diff
@@ -89,6 +122,41 @@ func main() {
 		log.Fatalf("failed to parse configuration passed to the program arguments: %v", err)
 	}
 
+	// TODO: investigate how we can unify logger
```

Comment:
Can you make GH issue for that?

Reply:
https://github.com/openshift-service-mesh/federation/issues/130